.github/workflows/tests.yaml

```diff
@@ -10,6 +10,8 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    env:
+      DRAND_TEST_LOGS: "${{ runner.debug == '1' && 'DEBUG' || 'INFO' }}"
```

Comment:
perhaps I'm being dense, but what is `runner.debug == '1'` referring to? I couldn't see anything in github action docs

Reply:
I'm assuming it's what's stopping you extracting the env to the root of the file so it's shared between everything

Reply:
https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context

But it seems it's not available there yet. So I'll try with the `RUNNER_DEBUG` env variable I guess.

Reply:
damned half-baked github features :<